bin/setup_dev

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 
-pushd `dirname "$0"`/../docker
-docker-compose up
-docker-compose stop
+APP_ROOT=`dirname "$0"`/..
+BIN_DIR="$APP_ROOT"/bin
+
+pushd "$APP_ROOT"/docker
+docker-compose build
+$BIN_DIR/run_docker bundle install
```

Comment:
For me it is not working, if called from project's root folder.

I have modified it locally to make it work to something like this:

``` sh
pushd "$APP_ROOT"/docker
docker-compose build
popd
pushd "$BIN_DIR"
./run_docker bundle install
./run_docker bundle exec rake db:create
./run_docker bundle exec rake db:migrate
./run_docker bundle exec rake rswag:specs:swaggerize
popd
```

Reply:
The scripts are now changed to be able to run called from anywhere.